projects/packages/my-jetpack/_inc/hooks/use-notification-watcher/use-protect-threats-detected-notice.tsx

```diff
@@ -43,9 +42,9 @@ const useProtectThreatsDetectedNotice = ( redBubbleAlerts: RedBubbleAlerts ) =>
 
 	const onCloseClick = useCallback( () => {
 		createCookie( 'protect_threats_detected_dismissed', 7 );
-		delete redBubbleAlerts.protect_has_threats;
+		delete redBubbleAlerts?.protect_has_threats;
```

Comment:
Is it necessary and works well now after migration to react-query.

For sure, it is a bad pattern we could solve, now or in the near future.

Reply:
Yes I agree, there are a lot of ways we could improve how the red bubble slugs on the frontend. This project being about moving data from backend blocking http requests to async loading on the frontend, I'm not sure it's the best place to address all these improvements 🤔 Especially seeing as how large this PR is already. I could handle it in a followup task, but I'm still unsure of how well it fits in the scope of this project. On top of the PR being large, this project as a whole is pretty large and I want to be cautious of how many things it covers to prevent it from going on too long. Does that sound reasonable?